### PR TITLE
dev

### PR DESCRIPTION
**Pull request by DominiqueMakowski**

### CODE REVIEW

**Comment by DominiqueMakowski** [comment]
And on the side, a facetted plot of the evolution of indices of significance:

[](https://www.datanovia.com/en/blog/gganimate-how-to-create-plots-with-beautiful-animation-in-r/)

We could then think about having multiple "evidence accumulation plots" depending on parameters SUCH AS prior specification (we could be cool for papers, as supp material or what)

**Comment by strengejacke** [comment]
?

**Comment by DominiqueMakowski** [comment]
lol I am trying to create a fancy plot that we could use at least in the README and vignettes 😁

**Comment by mattansb** [comment]
That is so cool!!!! We should use that in the priors paper!

**Comment by DominiqueMakowski** [comment]
Thanks @mattansb for a more enthusiastic comment than a depressing "?" 😬

**Comment by strengejacke** [comment]
:scream:
I just successfully reduced package size ~1mb, and now it's being filled with strange animations.

**Comment by DominiqueMakowski** [comment]
okay okay I'll move it to easystats/easystats 😠 😄

**Comment by DominiqueMakowski** [comment]
PS: @mattansb in print.bf_parameters, I swapped the column name as `"Bayes factor"` in favour of `BF` so its consistent with the actual name of the column (just now I got confused by trying to pull a column named "Bayes factor") (and to avoid redundancy as "Bayes Factor" is in the header)

**Comment by mattansb** [comment]
@DominiqueMakowski then I think you might need to also change it in the other `print` method...?

**Comment by DominiqueMakowski** [comment]
which other?

File: R/print.bayesfactor_parameters.R [hunk 1]
```diff
@@ -10,7 +10,7 @@ print.bayesfactor_parameters <- function(x, digits = 2, log = FALSE, ...) {
 
   BFE$BF <- .format_big_small(BFE$BF, digits = digits)
 
-  colnames(BFE)[colnames(BFE) == "BF"] <- "Bayes Factor"
+  colnames(BFE)[colnames(BFE) == "BF"] <- "BF"
```
Comment (strengejacke) [inline R/print.bayesfactor_parameters.R]: I think we don't need code line at all in this case?

Comment (DominiqueMakowski) [reply R/print.bayesfactor_parameters.R]: actually, yes

**Comment by mattansb** [comment]
@DominiqueMakowski post this gif to twitter!! It's sooooo cooooollll!

**Comment by DominiqueMakowski** [comment]
I am waiting for the final fixed version (it takes ages to compute 😅)

**Comment by DominiqueMakowski** [comment]
Here it is with 60 participants :) 

Can we merge this?

**Comment by mattansb** [comment]
Merge to Twitter? ✔

**Comment by DominiqueMakowski** [comment]
(merged to twitter 😁) Now the PR?

**Pull request merged**